ckanext/dalrrd_emc_dcpr/plugin.py

```diff
@@ -15,3 +20,85 @@ def update_config(self, config_):
 
     def get_commands(self):
         return [test_ckan_cmd]
+
+    # IDatasetForm
+
+    def _admins_only_create(self, key, data, errors, context):
```

Comment:
Whenever you implement a class method that does not make use of the `self` parameter, please consider writing it as a standalone function instead. This makes it easier to test the code.